cms-plugin/src/main/java/com/agiletec/plugins/jacms/aps/system/services/resource/ResourceService.java

```diff
@@ -46,6 +46,8 @@ public class ResourceService implements IResourceService,
 
     private final EntLogger logger = EntLogFactory.getSanitizedLogger(this.getClass());
     
+    public static final String CORRELATION_CODE_PARAMETER = "cc";
```

Comment:
this is a duplicated constant pls use `ResourcesController.ID_NAME_OF_CORRELATION_CODE`